source/Sashimi.Terraform.Tests/TerraformValidatorFixture.cs

```diff
@@ -0,0 +1,251 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Autofac;
+using FluentAssertions;
+using FluentValidation.TestHelper;
+using NUnit.Framework;
+using Octopus.Server.Extensibility.HostServices.Model;
+using Sashimi.Server.Contracts;
+using Sashimi.Server.Contracts.ActionHandlers.Validation;
+using Sashimi.Server.Contracts.CloudTemplates;
+using Sashimi.Terraform.Validation;
+using Sashimi.Tests.Shared.Server;
+
+namespace Sashimi.Terraform.Tests
+{
+    [TestFixture]
+    public class TerraformValidatorFixture
+    {
+        TerraformValidator? validator;
+        
+        [SetUp]
+        public void Setup() {
+            var builder = new ContainerBuilder();
+            builder.RegisterModule<TerraformModule>();
+            builder.RegisterModule<ServerModule>();
+            using (var container = builder.Build())
+            {
+                validator = new TerraformValidator(container.Resolve<ICloudTemplateHandlerFactory>());
+            }
+        }
+
+        [Test]
+        [TestCase(TerraformActionTypes.Apply)]
+        [TestCase(TerraformActionTypes.Destroy)]
+        public void Should_have_error_when_empty_inline_template(string actionType)
+        {
+            var result = validator.TestValidate(new DeploymentActionValidationContext(actionType,
+                new Dictionary<string, string>
+                {
+                    {KnownVariables.Action.Script.ScriptSource, KnownVariableValues.Action.Script.ScriptSource.Inline}
+                }, new List<PackageReference>()));
+            result.IsValid.Should().BeFalse();
+            result.Errors.Single().PropertyName.Should().Be(TerraformSpecialVariables.Action.Terraform.Template);
```

Comment:
I think you can use `ContainSingle` here:

`result.Errors.Should().ContainSingle().Which.PropertyName.Should().Be(TerraformSpecialVariables.Action.Terraform.Template);`

Reply:
fixed